src/redisWrapper.js

```diff
@@ -14,19 +14,22 @@ const COMPONENT_NAME = "/RedisWrapper";
 const VERROR_CLUSTER_NAME = "RedisWrapperError";
 
 const INTEGRATION_MODE = Object.freeze({
+  CF_REDIS_CLUSTER: "CF_REDIS_CLUSTER",
   CF_REDIS: "CF_REDIS",
   LOCAL_REDIS: "LOCAL_REDIS",
   NO_REDIS: "NO_REDIS",
 });
 
 const logger = new Logger(COMPONENT_NAME);
+const redisCredentials = cfEnv.cfServiceCredentialsForLabel("redis-cache");
```

Comment:
not a fan of doing things like this top-level...

Reply:
I would wrap this into a function and the function can cache it top level --> so that `cfEnv.cfServiceCredentialsForLabel` is called lazy

Reply:
or even better I just saw that env in your case is also a class. So I would move it there. As env is a singleton this should be suitable for the purpose

Reply:
I usually agree and use lazy loading over top level execution, but the cf env stuff is fast and only relies on env variables and no real runtime state... if you don't have it here, you end up accessing the credentials in every place you use it. on sendCommand on createClient and in the integration mode, which may look a bit odd. 

I will code it out and see what looks better.